pkg/components/bmerevents/capabilities.go

```diff
@@ -1,4 +1,4 @@
-package telcoedgeztp
```

Comment:
Did you rename the file, or git just matched them? Lots of those renaming seem random. It is not really bmerevents replacing telcoedge, right?